ui/raidboss/popup-text.js

```diff
@@ -669,7 +669,7 @@ class PopupText {
 
     if ('promise' in trigger) {
       if (typeof trigger.promise === 'function') {
-        promise = trigger.promise(data, matches);
+        promise = trigger.promise(this.data, matches);
```

Comment:
I think the function this is in is bound weirdly and I never fixed it.  I think this may need to be s/this/that/?

Reply:
The scoping of this statement is the same as the scoping of the `let that = this;` statement above, so I'm not sure there would be a difference here? If the scoping does need to be adjusted, it would also need to be adjusted for a bunch of other things that are obviously currently working, such as below on line 696 `this.timers.push...`, or above on lines 657 or 503.

Reply:
Oh, ok.  I should have scrolled all the way up.  I think at one point this was rebound in the function and it got changed to be an arrow function.  I'll clean this up after this to avoid future confusion (on my part).